client/src/components/GroupCard/GroupCard.css

```diff
@@ -1,39 +1,80 @@
 .GroupCard {
-    border-radius: 3px;
-    color: var(--white);
-    display: flex;
-    flex-direction: column;
-    background-size: cover;
-    background-repeat: no-repeat;
-    text-decoration: none;
+  position: absolute;
+  border-radius: 3px;
+  color: var(--white);
+  display: flex;
+  flex-direction: column;
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-decoration: none;
+
+  width: 100%;
 }
 
 .GroupCard__text-container {
-    min-height: 150px;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1vh 0 2vh 0;
+  width: 100%;
+  /* border: 2px solid black; */
 }
 
 .GroupCard__group-name {
-    font-size: 1.5rem;
-    line-height: 90%;
-    padding: 2.5rem 0rem 1rem 1.2rem;
-    font-weight: var(--title-weight);
-    margin-top: 0;
+  font-size: 1.2vw;
+  /* line-height: 90%; */
+  padding: 0 0rem 0rem 1.2rem;
+  margin-top: 0vh;
+  margin-bottom: 6vh;
+  max-width: 100%;
+  max-height: 6vh;
+  font-weight: var(--title-weight);
 }
 
 .GroupCard__details {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-around;
-    font-size: 0.9rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  font-size: 0.9rem;
 }
 
 .GroupCard__members {
-    margin: 0.2rem;
+  margin: 0.2rem;
+  /* border: 2px solid black; */
 }
 
 .GroupCard__location {
-    margin: 0.2rem;
+  margin: 0.2rem;
+  /* border: 2px solid black; */
+}
+
+/* Large devices */
+@media only screen and (max-width: 992px) {
+  .GroupCard__group-name {
+    font-size: 1.4vw;
+  }
+  .GroupCard__details {
+  font-size: 1rem;
+  }
+}
+
+/* Medium devices */
+@media only screen and (max-width: 768px) {
+ .GroupCard__group-name {
+    font-size: 1.8vw;
```

Comment:
rem or px is probably better for font-size. Can you use one of those?

Reply:
If you need to change the value, you can do so in a media query

Reply:
@ZakkMan There is HomePage stuff in here, I started messing with stuff from my HomePage branch and then decided to make this new branch, but i've accounted for all of the changes in regards to that. I'll take a look at the font size 💯